### PR TITLE
Deploying core and additional services

### PR DESCRIPTION
**Pull request by tbazant**
This update extends the bootstrap cluster deployment and describes how to add new hosts to the cluster as well as deploy MONs

### CODE REVIEW

File: xml/deploy_cephadm.xml [hunk 1]
```diff
@@ -35,6 +35,11 @@
     Configure the basic properties of the cluster and deploy it.
    </para>
   </listitem>
+  <listitem>
+   <para>
+    Deploy core cluster services on more nodes
```
Comment (tbazant) [inline xml/deploy_cephadm.xml:40]:
```suggestion
    Deploy core cluster services to more nodes.
```

File: xml/deploy_cephadm.xml [hunk 2]
```diff
@@ -643,4 +648,276 @@ config: OK
    </tip>
   </sect2>
  </sect1>
+ <sect1 xml:id="deploy-cephadm-day2">
+  <title>Further Deployment and Administration (Day 2)</title>
+
+  <para>
+   After you have deployed the basic &ceph; cluster, you need to deploy core
+   services on more cluster nodes. To make the cluster data accessible to
```
Comment (tbazant) [inline xml/deploy_cephadm.xml:656]:
```suggestion
   services to more cluster nodes. To make the cluster data accessible to
```

File: xml/deploy_cephadm.xml [hunk 3]
```diff
@@ -737,6 +742,130 @@ mon.ses-min1        ses-min1 running (5d) 8m ago    12d 15.2.0.108 5bf12403d0bd
...
+    <para>
+     Replace <replaceable>HOST_SPECIFICATION</replaceable> with a string in the
+     form of
+    </para>
+<screen><replaceable>SHORT_HOST_NAME</replaceable>:<replaceable>IP_ADDRESS</replaceable></screen>
+    <para>
+     For example:
+    </para>
+<screen>&prompt.cephuser;ceph orch apply mon ses-min2:10.100.25.52</screen>
+   </sect3>
```
Comment (sebastian-philipp) [inline xml/deploy_cephadm.xml:865]: I think MGRs are also good without the `:ip` part

File: xml/deploy_cephadm.xml [hunk 4]
```diff
@@ -737,6 +742,130 @@ mon.ses-min1        ses-min1 running (5d) 8m ago    12d 15.2.0.108 5bf12403d0bd
...
+      </para>
+<screen>&prompt.cephuser;ceph config set mon public_network <replaceable>MON_SUBNET_IP</replaceable></screen>
+      <para>
+       For example:
+      </para>
+<screen>&prompt.cephuser;ceph config set mon public_network 10.1.2.0/24</screen>
+      <para>
+       Then replace <replaceable>HOST_SPECIFICATION</replaceable> with a list
+       of short host names of the MONs you need to add"
+      </para>
```
Comment (sebastian-philipp) [inline xml/deploy_cephadm.xml:833]: Hm. actually ceph-salt should create an environment where this is not required.

**Pull request merged**